verify/verifier.go

```diff
@@ -183,11 +191,26 @@ func (v *Verifier) verifySecurityParams(blobHeader BlobHeader, batchHeader *disp
 	}
 
 	// ensure that required quorums are present in the confirmed ones
-	for _, quorum := range v.cv.quorumsRequired {
+	for _, quorum := range requiredQuorum(int64(batchHeader.ReferenceBlockNumber), v) {
```

Comment:
Why cast every-time vs taking a `uint32` as input to `requiredQuorum`?

Reply:
I think I left it after rebasing this branch from the one removing the archive nodes necessity.
I agree, is unnecessary. I'll change it